site/pages/gc-form-en.hbs

```diff
@@ -0,0 +1,197 @@
+		&#x3C;label for=&#x22;checkbox_3&#x22;&#x3E;Option 3&#x3C;/label&#x3E;
+	&#x3C;/li&#x3E;
+	&#x3C;li class=&#x22;checkbox&#x22;&#x3E;
+		&#x3C;input type=&#x22;checkbox&#x22; id=&#x22;checkbox_4&#x22;&#x3E;
+		&#x3C;label for=&#x22;checkbox_4&#x22;&#x3E;Option 4&#x3C;/label&#x3E;
+	&#x3C;/li&#x3E;
+&#x3C;/ul&#x3E;
+
+<span class="com">// Small checkboxes:</span>&#x3C;ul class=&#x22;list-unstyled gc-form&#x22;&#x3E;
+	&#x3C;li class=&#x22;checkbox checkbox-small&#x22;&#x3E;
+		&#x3C;input type=&#x22;checkbox&#x22; id=&#x22;confirmation-checkbox&#x22;&#x3E;
+		&#x3C;label for=&#x22;confirmation-checkbox&#x22;&#x3E;Single option&#x3C;/label&#x3E;
+	&#x3C;/li&#x3E;
+&#x3C;/ul&#x3E;</code></pre>
+		</div>
+	</div>
+	<h3>Radio buttons</h3>
+	<div class="row">
+		<div class="col-md-4">
+			<div class="panel panel-default">
+				<div class="panel-body">
+					<h5 class="mrgn-tp-0">Appearance</h5>
+					<h6>Radio buttons</h6>
+					<ul class="list-unstyled gc-form">
+						<li class="radio">
+							<input id="radio-option-1" type="radio" name="radio-example">
+							<label for="radio-option-1">Option 1</label>
+						</li>
+						<li class="radio">
+							<input id="radio-option-2" type="radio" name="radio-example">
+							<label for="radio-option-2">Option 2</label>
+						</li>
+						<li class="radio">
+							<input id="radio-option-3" type="radio" name="radio-example">
+							<label for="radio-option-3">Option 3</label>
+						</li>
+					</ul>
+				</div>
+			</div>
+		</div>
+		<div class="col-md-4">
+			<h5 class="mrgn-tp-0 text-success"><span class="glyphicon glyphicon-ok-circle"></span> Correct use</h5>
+			<p>Compliance point(s):</p>
+			<ul>
+				<li>Use radio buttons when only one selection can be made.</li>
+			</ul>
+			<h5 class="mrgn-tp-0 text-danger"><span class="glyphicon glyphicon-remove-circle"></span> Incorrect use</h5>
+			<p>Compliance point(s):</p>
+			<ul>
+				<li>Do not use if there is a need for users to select more than one option</li>
+			</ul>
+		</div>
+		<div class="col-md-4">
+			<h5 class="mrgn-tp-0">Code</h5>
+			<pre class="prettyprint prettyprinted" style=""><code><span class="com">// Radio buttons:</span>&#x3C;ul class=&#x22;list-unstyled gc-form&#x22;&#x3E;
+	&#x3C;li class=&#x22;radio&#x22;&#x3E;
+		&#x3C;input id=&#x22;radio-option-1&#x22; type=&#x22;radio&#x22; name=&#x22;radio-example&#x22;&#x3E;
+		&#x3C;label for=&#x22;radio-option-1&#x22;&#x3E;Option 1&#x3C;/label&#x3E;
+	&#x3C;/li&#x3E;
+	&#x3C;li class=&#x22;radio&#x22;&#x3E;
+		&#x3C;input id=&#x22;radio-option-2&#x22; type=&#x22;radio&#x22; name=&#x22;radio-example&#x22;&#x3E;
+		&#x3C;label for=&#x22;radio-option-2&#x22;&#x3E;Option 2&#x3C;/label&#x3E;
+	&#x3C;/li&#x3E;
+	&#x3C;li class=&#x22;radio&#x22;&#x3E;
+		&#x3C;input id=&#x22;radio-option-3&#x22; type=&#x22;radio&#x22; name=&#x22;radio-example&#x22;&#x3E;
+		&#x3C;label for=&#x22;radio-option-3&#x22;&#x3E;Option 3&#x3C;/label&#x3E;
+	&#x3C;/li&#x3E;
+&#x3C;/ul&#x3E;</code></pre>
```

Comment:
Not necessary an issue, more a comment. Usually we encode the character "<" ```&lt;``` and the character ">" ```&gt;``` instead of using the hexadecimal entities number. Double quote don't need to be encoded, but when we do it is like ```&quot;```

Reply:
Based on the html entity encoding software I was using. If there's a recommended tool for this, I'd be happy to use that instead. Suggestions?

Reply:
Sometime that could be just a setting in your editing tools. But as I said that is not an issue. It is just uncommon to encode the character greater than and lower than by using their corresponding hexadecimal entities.

Reply:
I'm using a browser based software: mothereff.in/html-entities. Was the first thing I found when searching about how to do those code snippets...

Anyway, if it's not much of an issue, then I'll just leave this one as is for convenience.